python3Packages.jax: 0.8.1 -> 0.8.2

GaetanLepage opened the pull request:
<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

- **python3Packages.jaxlib: format prefetch script**
- **python3Packages.jax: 0.8.1 -> 0.8.2**

Diff: https://github.com/google/jax/compare/jax-v0.8.1...jax-v0.8.2
Changelog: https://docs.jax.dev/en/latest/changelog.html

- Built on platform:
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [x] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

GaetanLepage committed:
python3Packages.jax: 0.8.1 -> 0.8.2

Diff: https://github.com/google/jax/compare/jax-v0.8.1...jax-v0.8.2

Changelog: https://docs.jax.dev/en/latest/changelog.html

GaetanLepage committed:
python3Packages.jaxlib: format prefetch script

samuela reviewed (approved):
diff lgtm, thanks @GaetanLepage !

GaetanLepage committed:
python3Packages.chex: skip failing tests

https://github.com/google-deepmind/chex/issues/422

GaetanLepage committed:
python3Packages.equinox: fix jax 0.8.2 compatibility

https://github.com/patrick-kidger/equinox/pull/1162

GaetanLepage commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 472927`
Commit: `8b84aa899f8140b268fdd474117ad3c42295871d`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 240 packages built:</summary>
  <ul>
    <li>exo</li>
    <li>exo.dist</li>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.anndata</li>
    <li>python312Packages.anndata.dist</li>
    <li>python312Packages.array-api-compat</li>
    <li>python312Packages.array-api-compat.dist</li>
    <li>python312Packages.augmax</li>
    <li>python312Packages.augmax.dist</li>
    <li>python312Packages.blackjax</li>
    <li>python312Packages.blackjax.dist</li>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
    <li>python312Packages.chex</li>
    <li>python312Packages.chex.dist</li>
    <li>python312Packages.clu</li>
    <li>python312Packages.clu.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.deepface</li>
    <li>python312Packages.deepface.dist</li>
    <li>python312Packages.dm-haiku</li>
    <li>python312Packages.dm-haiku.dist</li>
    <li>python312Packages.dm-sonnet</li>
    <li>python312Packages.dm-sonnet.dist</li>
    <li>python312Packages.edward</li>
    <li>python312Packages.edward.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.flowmc</li>
    <li>python312Packages.flowmc.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.jax</li>
    <li>python312Packages.jax-cuda12-pjrt</li>
    <li>python312Packages.jax-cuda12-plugin</li>
    <li>python312Packages.jax.dist</li>
    <li>python312Packages.jaxlib (python312Packages.jaxlib-bin)</li>
    <li>python312Packages.jaxlib.dist (python312Packages.jaxlib-bin.dist)</li>
    <li>python312Packages.jaxopt</li>
    <li>python312Packages.jaxopt.dist</li>
    <li>python312Packages.jmp</li>
    <li>python312Packages.jmp.dist</li>
    <li>python312Packages.keras</li>
    <li>python312Packages.keras.dist</li>
    <li>python312Packages.kserve</li>
    <li>python312Packages.kserve.dist</li>
    <li>python312Packages.lerobot</li>
    <li>python312Packages.lerobot.dist</li>
    <li>python312Packages.lineax</li>
    <li>python312Packages.lineax.dist</li>
    <li>python312Packages.mhcflurry</li>
    <li>python312Packages.mhcflurry.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.mtcnn</li>
    <li>python312Packages.mtcnn.dist</li>
    <li>python312Packages.mujoco-mjx</li>
    <li>python312Packages.mujoco-mjx.dist</li>
    <li>python312Packages.nifty8</li>
    <li>python312Packages.nifty8.dist</li>
    <li>python312Packages.optax</li>
    <li>python312Packages.optax.dist</li>
    <li>python312Packages.optax.testsout</li>
    <li>python312Packages.optimistix</li>
    <li>python312Packages.optimistix.dist</li>
    <li>python312Packages.orbax-checkpoint</li>
    <li>python312Packages.orbax-checkpoint.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.paramax</li>
    <li>python312Packages.paramax.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.pymanopt</li>
    <li>python312Packages.pymanopt.dist</li>
    <li>python312Packages.pyvers</li>
    <li>python312Packages.pyvers.dist</li>
    <li>python312Packages.retinaface</li>
    <li>python312Packages.retinaface.dist</li>
    <li>python312Packages.scanpy</li>
    <li>python312Packages.scanpy.dist</li>
    <li>python312Packages.scikit-bio</li>
    <li>python312Packages.scikit-bio.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.tensordict</li>
    <li>python312Packages.tensordict.dist</li>
    <li>python312Packages.tensorflow-datasets</li>
    <li>python312Packages.tensorflow-datasets.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.tinygrad</li>
    <li>python312Packages.tinygrad.dist</li>
    <li>python312Packages.torchrl</li>
    <li>python312Packages.torchrl.dist</li>
    <li>python312Packages.treescope</li>
    <li>python312Packages.treescope.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>vllm (python312Packages.vllm)</li>
    <li>vllm.dist (python312Packages.vllm.dist)</li>
    <li>python312Packages.vmas</li>
    <li>python312Packages.vmas.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.clu</li>
    <li>python313Packages.clu.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.dm-sonnet</li>
    <li>python313Packages.dm-sonnet.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax-cuda12-pjrt</li>
    <li>python313Packages.jax-cuda12-plugin</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.kserve</li>
    <li>python313Packages.kserve.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.mhcflurry</li>
    <li>python313Packages.mhcflurry.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.mujoco-mjx.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.scanpy</li>
    <li>python313Packages.scanpy.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.tensorflow-datasets</li>
    <li>python313Packages.tensorflow-datasets.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.tinygrad</li>
    <li>python313Packages.tinygrad.dist</li>
    <li>python313Packages.torchrl</li>
    <li>python313Packages.torchrl.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
    <li>python313Packages.tyro</li>
    <li>python313Packages.tyro.dist</li>
    <li>python313Packages.unsloth</li>
    <li>python313Packages.unsloth-zoo</li>
    <li>python313Packages.unsloth-zoo.dist</li>
    <li>python313Packages.unsloth.dist</li>
    <li>python313Packages.vllm</li>
    <li>python313Packages.vllm.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 206 packages built:</summary>
  <ul>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python312Packages.ale-py</li>
    <li>python312Packages.ale-py.dist</li>
    <li>python312Packages.anndata</li>
    <li>python312Packages.anndata.dist</li>
    <li>python312Packages.array-api-compat</li>
    <li>python312Packages.array-api-compat.dist</li>
    <li>python312Packages.augmax</li>
    <li>python312Packages.augmax.dist</li>
    <li>python312Packages.blackjax</li>
    <li>python312Packages.blackjax.dist</li>
    <li>python312Packages.brax</li>
    <li>python312Packages.brax.dist</li>
    <li>python312Packages.chex</li>
    <li>python312Packages.chex.dist</li>
    <li>python312Packages.dalle-mini</li>
    <li>python312Packages.dalle-mini.dist</li>
    <li>python312Packages.deepface</li>
    <li>python312Packages.deepface.dist</li>
    <li>python312Packages.dm-haiku</li>
    <li>python312Packages.dm-haiku.dist</li>
    <li>python312Packages.edward</li>
    <li>python312Packages.edward.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.evosax</li>
    <li>python312Packages.evosax.dist</li>
    <li>python312Packages.flax</li>
    <li>python312Packages.flax.dist</li>
    <li>python312Packages.flowmc</li>
    <li>python312Packages.flowmc.dist</li>
    <li>python312Packages.gymnasium</li>
    <li>python312Packages.gymnasium.dist</li>
    <li>python312Packages.jax</li>
    <li>python312Packages.jax-cuda12-pjrt</li>
    <li>python312Packages.jax-cuda12-plugin</li>
    <li>python312Packages.jax.dist</li>
    <li>python312Packages.jaxlib (python312Packages.jaxlib-bin)</li>
    <li>python312Packages.jaxlib.dist (python312Packages.jaxlib-bin.dist)</li>
    <li>python312Packages.jaxopt</li>
    <li>python312Packages.jaxopt.dist</li>
    <li>python312Packages.jmp</li>
    <li>python312Packages.jmp.dist</li>
    <li>python312Packages.keras</li>
    <li>python312Packages.keras.dist</li>
    <li>python312Packages.lerobot</li>
    <li>python312Packages.lerobot.dist</li>
    <li>python312Packages.lineax</li>
    <li>python312Packages.lineax.dist</li>
    <li>python312Packages.mhcflurry</li>
    <li>python312Packages.mhcflurry.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.mtcnn</li>
    <li>python312Packages.mtcnn.dist</li>
    <li>python312Packages.mujoco-mjx</li>
    <li>python312Packages.mujoco-mjx.dist</li>
    <li>python312Packages.nifty8</li>
    <li>python312Packages.nifty8.dist</li>
    <li>python312Packages.optax</li>
    <li>python312Packages.optax.dist</li>
    <li>python312Packages.optax.testsout</li>
    <li>python312Packages.optimistix</li>
    <li>python312Packages.optimistix.dist</li>
    <li>python312Packages.orbax-checkpoint</li>
    <li>python312Packages.orbax-checkpoint.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.paramax</li>
    <li>python312Packages.paramax.dist</li>
    <li>python312Packages.pettingzoo</li>
    <li>python312Packages.pettingzoo.dist</li>
    <li>python312Packages.pymanopt</li>
    <li>python312Packages.pymanopt.dist</li>
    <li>python312Packages.pyvers</li>
    <li>python312Packages.pyvers.dist</li>
    <li>python312Packages.retinaface</li>
    <li>python312Packages.retinaface.dist</li>
    <li>python312Packages.scikit-bio</li>
    <li>python312Packages.scikit-bio.dist</li>
    <li>python312Packages.shimmy</li>
    <li>python312Packages.shimmy.dist</li>
    <li>python312Packages.skrl</li>
    <li>python312Packages.skrl.dist</li>
    <li>python312Packages.stable-baselines3</li>
    <li>python312Packages.stable-baselines3.dist</li>
    <li>python312Packages.tensordict</li>
    <li>python312Packages.tensordict.dist</li>
    <li>python312Packages.tianshou</li>
    <li>python312Packages.tianshou.dist</li>
    <li>python312Packages.treescope</li>
    <li>python312Packages.treescope.dist</li>
    <li>python312Packages.tyro</li>
    <li>python312Packages.tyro.dist</li>
    <li>python312Packages.unsloth</li>
    <li>python312Packages.unsloth-zoo</li>
    <li>python312Packages.unsloth-zoo.dist</li>
    <li>python312Packages.unsloth.dist</li>
    <li>python312Packages.vmas</li>
    <li>python312Packages.vmas.dist</li>
    <li>python312Packages.vqgan-jax</li>
    <li>python312Packages.vqgan-jax.dist</li>
    <li>python312Packages.waymax</li>
    <li>python312Packages.waymax.dist</li>
    <li>python313Packages.ale-py</li>
    <li>python313Packages.ale-py.dist</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.brax</li>
    <li>python313Packages.brax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.dalle-mini</li>
    <li>python313Packages.dalle-mini.dist</li>
    <li>python313Packages.deepface</li>
    <li>python313Packages.deepface.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.edward</li>
    <li>python313Packages.edward.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.evosax</li>
    <li>python313Packages.evosax.dist</li>
    <li>python313Packages.flax</li>
    <li>python313Packages.flax.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.gymnasium</li>
    <li>python313Packages.gymnasium.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax-cuda12-pjrt</li>
    <li>python313Packages.jax-cuda12-plugin</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.keras</li>
    <li>python313Packages.keras.dist</li>
    <li>python313Packages.lerobot</li>
    <li>python313Packages.lerobot.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.mhcflurry</li>
    <li>python313Packages.mhcflurry.dist</li>
    <li>python313Packages.mtcnn</li>
    <li>python313Packages.mtcnn.dist</li>
    <li>python313Packages.mujoco-mjx</li>
    <li>python313Packages.mujoco-mjx.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pettingzoo</li>
    <li>python313Packages.pettingzoo.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.retinaface</li>
    <li>python313Packages.retinaface.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.shimmy</li>
    <li>python313Packages.shimmy.dist</li>
    <li>python313Packages.skrl</li>
    <li>python313Packages.skrl.dist</li>
    <li>python313Packages.stable-baselines3</li>
    <li>python313Packages.stable-baselines3.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.tianshou</li>
    <li>python313Packages.tianshou.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
    <li>python313Packages.tyro</li>
    <li>python313Packages.tyro.dist</li>
    <li>python313Packages.unsloth</li>
    <li>python313Packages.unsloth-zoo</li>
    <li>python313Packages.unsloth-zoo.dist</li>
    <li>python313Packages.unsloth.dist</li>
    <li>python313Packages.vmas</li>
    <li>python313Packages.vmas.dist</li>
    <li>python313Packages.vqgan-jax</li>
    <li>python313Packages.vqgan-jax.dist</li>
    <li>python313Packages.waymax</li>
    <li>python313Packages.waymax.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 102 packages built:</summary>
  <ul>
    <li>minari (python313Packages.minari)</li>
    <li>minari.dist (python313Packages.minari.dist)</li>
    <li>python312Packages.anndata</li>
    <li>python312Packages.anndata.dist</li>
    <li>python312Packages.array-api-compat</li>
    <li>python312Packages.array-api-compat.dist</li>
    <li>python312Packages.augmax</li>
    <li>python312Packages.augmax.dist</li>
    <li>python312Packages.blackjax</li>
    <li>python312Packages.blackjax.dist</li>
    <li>python312Packages.chex</li>
    <li>python312Packages.chex.dist</li>
    <li>python312Packages.dm-haiku</li>
    <li>python312Packages.dm-haiku.dist</li>
    <li>python312Packages.equinox</li>
    <li>python312Packages.equinox.dist</li>
    <li>python312Packages.flowmc</li>
    <li>python312Packages.flowmc.dist</li>
    <li>python312Packages.jax</li>
    <li>python312Packages.jax.dist</li>
    <li>python312Packages.jaxlib (python312Packages.jaxlib-bin)</li>
    <li>python312Packages.jaxlib.dist (python312Packages.jaxlib-bin.dist)</li>
    <li>python312Packages.jaxopt</li>
    <li>python312Packages.jaxopt.dist</li>
    <li>python312Packages.jmp</li>
    <li>python312Packages.jmp.dist</li>
    <li>python312Packages.lineax</li>
    <li>python312Packages.lineax.dist</li>
    <li>python312Packages.minari</li>
    <li>python312Packages.minari.dist</li>
    <li>python312Packages.nifty8</li>
    <li>python312Packages.nifty8.dist</li>
    <li>python312Packages.optax</li>
    <li>python312Packages.optax.dist</li>
    <li>python312Packages.optax.testsout</li>
    <li>python312Packages.optimistix</li>
    <li>python312Packages.optimistix.dist</li>
    <li>python312Packages.orbax-checkpoint</li>
    <li>python312Packages.orbax-checkpoint.dist</li>
    <li>python312Packages.outlines</li>
    <li>python312Packages.outlines.dist</li>
    <li>python312Packages.paramax</li>
    <li>python312Packages.paramax.dist</li>
    <li>python312Packages.pymanopt</li>
    <li>python312Packages.pymanopt.dist</li>
    <li>python312Packages.pyvers</li>
    <li>python312Packages.pyvers.dist</li>
    <li>python312Packages.scikit-bio</li>
    <li>python312Packages.scikit-bio.dist</li>
    <li>python312Packages.tensordict</li>
    <li>python312Packages.tensordict.dist</li>
    <li>python312Packages.treescope</li>
    <li>python312Packages.treescope.dist</li>
    <li>python313Packages.anndata</li>
    <li>python313Packages.anndata.dist</li>
    <li>python313Packages.array-api-compat</li>
    <li>python313Packages.array-api-compat.dist</li>
    <li>python313Packages.augmax</li>
    <li>python313Packages.augmax.dist</li>
    <li>python313Packages.blackjax</li>
    <li>python313Packages.blackjax.dist</li>
    <li>python313Packages.chex</li>
    <li>python313Packages.chex.dist</li>
    <li>python313Packages.dm-haiku</li>
    <li>python313Packages.dm-haiku.dist</li>
    <li>python313Packages.equinox</li>
    <li>python313Packages.equinox.dist</li>
    <li>python313Packages.flowmc</li>
    <li>python313Packages.flowmc.dist</li>
    <li>python313Packages.jax</li>
    <li>python313Packages.jax.dist</li>
    <li>python313Packages.jaxlib (python313Packages.jaxlib-bin)</li>
    <li>python313Packages.jaxlib.dist (python313Packages.jaxlib-bin.dist)</li>
    <li>python313Packages.jaxopt</li>
    <li>python313Packages.jaxopt.dist</li>
    <li>python313Packages.jmp</li>
    <li>python313Packages.jmp.dist</li>
    <li>python313Packages.lineax</li>
    <li>python313Packages.lineax.dist</li>
    <li>python313Packages.nifty8</li>
    <li>python313Packages.nifty8.dist</li>
    <li>python313Packages.optax</li>
    <li>python313Packages.optax.dist</li>
    <li>python313Packages.optax.testsout</li>
    <li>python313Packages.optimistix</li>
    <li>python313Packages.optimistix.dist</li>
    <li>python313Packages.orbax-checkpoint</li>
    <li>python313Packages.orbax-checkpoint.dist</li>
    <li>python313Packages.outlines</li>
    <li>python313Packages.outlines.dist</li>
    <li>python313Packages.paramax</li>
    <li>python313Packages.paramax.dist</li>
    <li>python313Packages.pymanopt</li>
    <li>python313Packages.pymanopt.dist</li>
    <li>python313Packages.pyvers</li>
    <li>python313Packages.pyvers.dist</li>
    <li>python313Packages.scikit-bio</li>
    <li>python313Packages.scikit-bio.dist</li>
    <li>python313Packages.tensordict</li>
    <li>python313Packages.tensordict.dist</li>
    <li>python313Packages.treescope</li>
    <li>python313Packages.treescope.dist</li>
  </ul>
</details>

GaetanLepage committed:
python3Packages.array-api-compat: 1.12 -> 1.13

Diff: https://github.com/data-apis/array-api-compat/compare/1.12...1.13

Changelog: https://github.com/data-apis/array-api-compat/releases/tag/1.13
